### PR TITLE
SAK-45406 CM SampleDataLoader not loading because of exceptions

### PR DESCRIPTION
**Pull request by ern**

### CODE REVIEW

**Comment by ern** [comment]
The issue with SampleDataLoader not finishing is fixed, however I've uncovered some other issues that were muted.
Making this as a Draft for now as I've got more to fix.

**Comment by jonespm** [comment]
Thank for looking at it, looks like good changes! Was going to test this but it doesn't compile yet. I'll wait for it to be moved out of draft.

**Comment by ern** [comment]
Yeah it's not ready just yet ;)

File: edu-services/cm-service/cm-impl/hibernate-impl/impl/src/java/org/sakaiproject/coursemanagement/impl/CourseManagementServiceHibernateImpl.java [hunk 1]
```diff
@@ -471,13 +464,16 @@ public String getSectionCategoryDescription(String categoryCode) {
...
 	public String getEnrollmentStatusDescription(String statusId) {
-		return new ResourceLoader("enrollmentstatus").getString(statusId, statusId);
+		return enrollmentsMessages.getString(statusId, statusId);
 	}
 
 	public Map<String, String> getEnrollmentStatusDescriptions(Locale locale) {
-
-		ResourceLoader rl = new ResourceLoader("enrollmentstatus");
-		return ((Set<String>) rl.keySet()).stream().collect(Collectors.toMap(k -> k, k -> rl.getString(k, k)));
+		enrollmentsMessages.setContextLocale(locale);
```
Comment (jonespm) [inline edu-services/cm-service/cm-impl/hibernate-impl/impl/src/java/org/sakaiproject/coursemanagement/impl/CourseManagementServiceHibernateImpl.java:471]: Nice, another bug fixes in the original code, unused locale.

**Review by jonespm — APPROVED** [review]
All of these changes look great to me, removing unnecessary classes, methods to statics and modernizations. The sample data is loaded for me on an empty database and everything is working good. 

```
29-Apr-2021 18:40:25.238 INFO [main] org.sakaiproject.coursemanagement.impl.SampleDataLoader.load Finished loading sample CM data
```

File: edu-services/cm-service/cm-impl/hibernate-impl/impl/src/java/org/sakaiproject/coursemanagement/impl/CourseManagementServiceHibernateImpl.java (hunk 1 above)
Comment (ern) [reply edu-services/cm-service/cm-impl/hibernate-impl/impl/src/java/org/sakaiproject/coursemanagement/impl/CourseManagementServiceHibernateImpl.java:471]: locale can be null there

**Pull request merged**